modules/environment/init.zsh

```diff
@@ -33,7 +33,21 @@ unsetopt BG_NICE
 # Send the HUP signal to running jobs when the shell exits.
 unsetopt HUP
 
-# Report the status of background and suspended jobs before exiting a shell with job control; 
-# a second attempt to exit the shell will succeed. 
-# NO_CHECK_JOBS is best used only in combination with NO_HUP, else such jobs will be killed automatically
+# Report the status of background and suspended jobs before exiting a shell with job control;
+# a second attempt to exit the shell will succeed.
+# NO_CHECK_JOBS is best used only in combination with NO_HUP, else such jobs will be killed automatically.
 unsetopt CHECK_JOBS
+
+# Set less as the default pager.
+[[ -z ${PAGER} && ${+commands[less]} -ne 0 ]] && export PAGER=less
+
+# Termcap for less.
+if [[ ${PAGER} == 'less' ]]; then
+  export LESS_TERMCAP_mb=$'\E[01;31m'     # Begins blinking.
+  export LESS_TERMCAP_md=$'\E[01;31m'     # Begins bold.
+  export LESS_TERMCAP_me=$'\E[0m'         # Ends mode.
+  export LESS_TERMCAP_se=$'\E[0m'         # Ends standout-mode.
+  export LESS_TERMCAP_so=$'\E[00;47;30m'  # Begins standout-mode.
```

Comment:
This is what I get with `$'\E[00;47;30m'`:

<img width="553" alt="screen shot 2016-10-28 at 7 27 40 am" src="https://cloud.githubusercontent.com/assets/4120606/19807570/46185fc2-9ce6-11e6-9363-2f6b6c126ccb.png">

And this is what I get with `$'\E[01;44;33m'`:

<img width="553" alt="screen shot 2016-10-28 at 7 28 02 am" src="https://cloud.githubusercontent.com/assets/4120606/19807591/5f0ed8e4-9ce6-11e6-9329-2b96f206a489.png">